src/main/java/ru/hh/school/entity/Resume.java

```diff
@@ -15,9 +15,11 @@
   // https://vladmihalcea.com/from-jpa-to-hibernates-legacy-and-enhanced-identifier-generators/
 
   @Id
-  @GeneratedValue(/* здесь место для вашего кода */)
+  @GeneratedValue(generator = "seq", strategy = GenerationType.SEQUENCE)
+  @SequenceGenerator(name = "seq", sequenceName = "resume_id_seq",  allocationSize = 10)
   private Integer id;
 
+  @Column(name = "description")
   private String description;
 
   Resume() {}
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.